nifti2/nifti2_io.c

```diff
@@ -9258,7 +9261,8 @@ static int rci_read_data(nifti_image * nim, int64_t * pivots, int64_t * prods,
 
       /* make sure things look good here */
       if( *pivots != 0 ){
-         fprintf(stderr,"** NIFTI rciRD: final pivot == %lld!\n", *pivots);
+         fprintf(stderr,"** NIFTI rciRD: final pivot == %" PRId64 "\n",
```

Comment:
Did you mean to remove the exclamation point?

Reply:
Yes, it seemed like a mistake to be there.  I am not a fan of programs yelling at the users.  :)

Reply:
BTW, way to pay attention to the little details...  :)